services-core/gradle.properties

```diff
@@ -1,4 +1,4 @@
-VERSION_NAME=4.5.0-SNAPSHOT
+VERSION_NAME=4.6.0-SNAPSHOT
```

Comment:
I noticed this when I was just releasing 4.6.0.
If you rebase this PR to master - you will not need this commit.